ibexa/commerce/5.0/config/packages/ibexa.yaml

```diff
@@ -120,6 +120,14 @@ ibexa:
 
             page_layout: "@ibexadesign/pagelayout.html.twig"
             user_content_type_identifier: ['user', 'customer', 'member']
+            notifier:
+                subscriptions:
+                    Ibexa\Contracts\User\Notification\UserPasswordReset:
+                        channels:
+                            - email
```

Comment:
One question - this doesn't seem to be used in `ibexa/user`. We seem to always send an email (based on what I saw in the PR). Is this correct? Are we even making use of this setting?